gen-compdocs/go.mod

```diff
@@ -4,42 +4,42 @@ go 1.16
 
 require (
 	github.com/golangplus/testing v0.0.0 // indirect
-	github.com/spf13/cobra v1.2.1
+	github.com/spf13/cobra v1.4.0
 	github.com/spf13/pflag v1.0.5
-	github.com/yuin/goldmark v1.4.0
+	github.com/yuin/goldmark v1.4.1
 	github.com/yuin/goldmark-highlighting v0.0.0-20200307114337-60d527fdb691
-	k8s.io/component-base v0.23.0
+	k8s.io/component-base v0.24.0
 	k8s.io/kubectl v0.0.0
-	k8s.io/kubernetes v0.0.0
+	k8s.io/kubernetes v0.0.0-00010101000000-000000000000
```

Comment:
Why are the trailing zeros included?

Reply:
I have no idea how it comes in or what does it mean. I was only doing 'go mod tidy', 'go mod download'. There were some complaints about 'k8s.io/kubernetes' package, so I removed the `k8s.io/kubernetes v0.0.0` line and let `go` figure it out. This replacement was done by Go lang.